docs/pages/connect-your-client/mcp.mdx

```diff
@@ -0,0 +1,139 @@
+[Standard Input/Output (stdio)](https://modelcontextprotocol.io/docs/concepts/transports#standard-input%2Foutput-stdio).
+
+Similarly to `tsh db connect`, the database access MCP server command requires
+you to pick the database user and name the MCP connections will use. In
+addition, you can also filter which database will be available as an MCP
+resource, either using `--query` or `--labels` flags.
+
+As for the current version, **you can only specify a single value for
+`--db-user` and `--db-name`**. All selected databases must have a database and
+a database user. Otherwise, the queries will fail. 
+
+<Admonition type="note" title="Permissions">
+  Given that language models can perform any query on your database, we
+  recommend that you select a database user with only the permissions you want
+  to allow the language models to perform. Creating a user with read-only
+  permissions will prevent accidental changes to your database.
+</Admonition>
+
+#### VS Code
+
+You can either add the Teleport MCP server to
+[your workspace](https://code.visualstudio.com/docs/copilot/chat/mcp-servers#_add-an-mcp-server-to-your-workspace)
+or to the [user settings](https://code.visualstudio.com/docs/copilot/chat/mcp-servers#_add-an-mcp-server-to-your-user-settings).
+
+```json
+{
+  "mcp": {
+    "servers": {
+      "teleport-databases": {
+        "command": "tsh",
+        "args": [
+          "mcp",
+          "db",
+          "--db-user=readonly",
+          "--db-name=postgres",
+          "--labels=\"env=dev\""
+        ]
+      }
+    }
+  }
+}
+```
+
+#### Claude Desktop
+
+Open the Claude Desktop configuration file.
+- macOS: `~/Library/Application Support/Claude/claude_desktop_config.json`
+- Windows: `%APPDATA%\Claude\claude_desktop_config.json`
+
+And update the contents with the Teleport MCP server configuration.
+
+```json
+{
+  "mcpServers": {
+    "teleport-detabases": {
+      "command": "tsh",
+      "args": [
+        "mcp",
+        "db",
+        "--db-user=readonly",
+        "--db-name=postgres",
+        "--labels=\"env=dev\""
+      ]
+    }
+  }
+}
+```
+
+After updating the configuration, you must restart the Claude Desktop app before
+using the newly added MCPs.
+
+### Step 2/2. Usage
+
+After updating your MCP client configuration, you can use the Teleport
+databases. Each database protocol will have its own tool to execute queries.
+Given that you might have multiple databases from the same protocol, the tool
+call must specify which Teleport database it will perform the query against.
+
+The Teleport database resources can be retrieved automatically by the clients
+(using the available tool) and are also available as MCP resources. Depending
+on which client you're using, you can set the target database beforehand. For
+example, on Claude Desktop, you can attach the resource to the conversation and
+ask the model to perform the queries on the attached database.
+
+### Troubleshooting
+
+TODO
```

Comment:
Scenarios/tasks to consider:
1. Expired session
2. Invalid database user, database name
3. Some other error cases?
4. How do you get the logs?